apps/studio/components/interfaces/Auth/AuthTemplatesValidation.tsx

```diff
@@ -30,7 +30,7 @@ const CONFIRMATION: FormSchema = {
     },
   },
   validationSchema: object().shape({
-    MAILER_SUBJECTS_CONFIRMATION: string().required('Subject heading is required.'),
+    MAILER_SUBJECTS_CONFIRMATION: string().required('Subject is required.'),
```

Comment:
Similarly, 'Subject heading' is now just 'Subject' in https://github.com/supabase/supabase/pull/40069